beetmoverscript/script.py

```diff
@@ -358,19 +454,19 @@ def list_bucket_objects(context, s3_resource, prefix):
 
 # move_beet {{{1
 async def move_beet(context, source, destinations, locale,
```

Comment:
Note to self: this seems like a s/`artifact_pretty_name`/`artifact_checksums_path`. Still unclear to me a bit why this is needed but it could be related to the `checksums_path` field in the manifest, defined https://github.com/mozilla-releng/beetmoverscript/pull/182/files#diff-98ba072d44cc67fadf38dc459a83d934R425 coming from transforms. I'm very reluctant to changing any of the existing code from beetmoverscript but this seems harmless at first glance.